### DISCUSSION TITLE
Powered Up Hub 6253628 not connecting to Pybricks

### DISCUSSION THREAD

**Discussion by Ad2876** [pybricks/support#589] [Q&A] [answered]
Hi,
I am trying to connect the powered up hub (got it with LEGO batmobile 76112) with the pybricks code webpage.
I can pair it to my computer using Bluetooth but when I go to the website in my browser and click on the Bluetooth icon, it never recognizes the hub.
Even though I have tried pairing using Edge, Chromium and Chrome browsers as suggested on various forums. I have enabled experimental flags but still the hub is not being recognised as an available device.

I turned on device log in Chrome and the bluetooth error is: No advertisement flags found.
Does anyone else have the same problem, if so how did you solve it?

**Comment by laurensvalk** [comment, marked as answer]
You have to install Pybricks on the hub first. [Here's how](https://pybricks.com/install/technic-boost-city/#installing-the-pybricks-firmware) to do it.

Or, if you prefer video:

[![Instructions](https://img.youtube.com/vi/QLL7EegBGr0/0.jpg)](https://www.youtube.com/watch?v=QLL7EegBGr0)

**Reply by Ad2876** [reply]
Hi, thanks for your prompt reply.
I tried this.
The light on the hub remains white and the pybricks code window keeps on saying 'scanning for devices'
They do not connect at all.....

**Reply by Ad2876** [reply]
Now I am getting the message:
The hub was disconnected before flashing was completed. Restart the hub and try again.

**Reply by laurensvalk** [reply]
If you follow the steps in the video, when do you see the white light?

In the video, you'll see that the hub starts off, then the light should blink pink as you press and hold the button for 5 seconds.

Does it look different on your hub?

**Reply by laurensvalk** [reply]
OK, so you found out how to start the installation. Does the installation problem persist?

**Reply by Ad2876** [reply]
Hi, 
Your advice worked wonders.
It successfully connected and ran a few sample python codes.

**Reply by brianvert** [reply]
I've had exactly the same problem getting my computer to connect to the PoweredUp Technic Hub (#88012).  When I try to pair it via Bluetooth (which is on), it just spins and fails to recognize the device, so I can't install the firmware in the first place.  I can connect to the hub via the PoweredUp iPhone app, but that doesn't seem to provide a way to install Pybricks scripts and it appears Bluetooth connections are not supported for Google Chrome in iOS.

**Reply by laurensvalk** [reply]
@brianvert, What color does the hub light blink when you try this? White or purple?

Please check the instructions or the video. To install the firmware, use the firmware button (not the bluetooth button).

**Comment by brianvert** [comment]
﻿
﻿Purple.

Brian

**Reply by laurensvalk** [reply]
And when it blinks purple, did you try the Bluetooth button or the firmware installation button?

**Comment by brianvert** [comment]
﻿Purple.

Brian



Brian

**Comment by brianvert** [comment]
﻿We ended up getting it to work on a different computer. Thanks for asking!

Brian



Brian
